app/assets/v2/js/activity.js

```diff
@@ -753,6 +758,24 @@ $(document).ready(function() {
     });
   };
 
+  // add emoji to comment
+  let textArea = '';
```

Comment:
how this code works when status is loaded? for example on profile activity.js and status.js are loaded and seems use the same var name.

Reply:
the code still works when both status.js and activity.js are loaded on the townsquare and profile page. I have searched for the var name and was unable to find a duplicate, however, i have renamed the variable name to a more precise name `activityCommentTextArea` 

screenshots: https://imgur.com/a/Upbk7M4